Core.ApplicationServices/IUserService.cs

```diff
@@ -15,7 +15,7 @@ public interface IUserService : IDisposable
 
         void UpdateUser(User user, bool? sendMailOnUpdate, int? scopedToOrganizationId);
         void IssueAdvisMail(User user, bool reminder, int orgId);
-        PasswordResetRequest IssuePasswordReset(User user, string subject, string content);
+        PasswordResetRequest IssuePasswordReset(User user, string subject, string content, bool newUI);
```

Comment:
I think this UI concern does not belong in the userService and should be pushed as far out to the edge of the backend as possible.

For example the UserService can have an IResetPasswordUrlStrategy injected, and then the v1 and v2 user controllers can each have a userService with respectively oldUiStrategy and newUiStrategy. 
It’s a bit of extra work, but leads to a cleaner design and also keeps the decision about old/new route out in the controller layer so it’s less work to root out references to the “2 UI’s” situation when we drop the old one.

Reply:
To future readers: We agreed to leave it as it is, as it is only a temporary solution, and therefore the added complexity might not be worth it.